tardis/opacities/macro_atom/util.py

```diff
@@ -4,14 +4,14 @@
 from tardis import constants as const
 from tardis.transport.montecarlo import njit_dict
 
-h_cgs = const.h.cgs.value
-c = const.c.to("cm/s").value
-kb = const.k_B.cgs.value
-inv_c2 = 1 / (c**2)
+H_CGS = const.h.cgs.value
```

Comment:
Are any of these constants used? They are defined in other locations in tardis as well.